blog/2024-04-26-license-questions.mdx

```diff
@@ -0,0 +1,68 @@
+---
+slug: license-questions
+title: License Questions
+authors: [parker]
+tags: [panel, license]
+hide_table_of_contents: true
+---
+
+As some of you know, people have been asking us questions about when you need a license or not.
+
+I asked for people to get me questions they wanted clarifications on and here are some answers. 
+
+## modification questions
+
+1.  Q: What are the plans for the theming part of Pelican Panel and what will the front end look like?
+
+    A: Using [Filament themes](https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme) placed in `/resources/css/filament` will work and don't need a license, 
+    and so long as no files that are included with the panel are modified in the process, you do not need a commercial license.
+
+:::info
+    A theme selector will eventually be implemented.
+:::
+:::warning
+    Themes are unable to change the layout of the panel, but plugins can.
+:::
+:::info definitions
+    Theme: The color and style of the panel. (Ex. button colors or fonts)  
+    Layout: The way the panel is organized. (Ex. where the cpu usage shows up.)
+    Panel Code: The files that the panel ship with.
+:::
+
+2.  Q: Will only styling be a thing, or is there also some sort of support for custom layouts and functionalities?
+
+    A: Custom layouts can be made with either plugins or by writing a custom frontend that uses the API to interact with the panel.
+
+3.  Q: Some users will probably use "public" themes for the panel (maybe smth like unix or so), if they are a public host, since they aren't the ones making the theme, do they need to pay the commercial license?
```

Comment:
```suggestion
3.  Q: Some users will probably use "public" themes for the panel (maybe something like unix or so), if they are a public host, since they aren't the ones making the theme, do they need to pay the commercial license?
```

Reply:
This is the question as asked. I am not editing other peoples words.